packages/core/admin/server/controllers/admin.js

```diff
@@ -45,11 +47,21 @@ module.exports = {
     const currentEnvironment = strapi.config.get('environment');
     const autoReload = strapi.config.get('autoReload', false);
     const strapiVersion = strapi.config.get('info.strapi', null);
+    const dependencies = strapi.config.get('info.dependencies', {});
     const nodeVersion = process.version;
     const communityEdition = !strapi.EE;
+    const useYarn = await exists(path.join(process.cwd(), 'yarn.lock'));
 
     return {
-      data: { currentEnvironment, autoReload, strapiVersion, nodeVersion, communityEdition },
+      data: {
+        currentEnvironment,
+        autoReload,
+        strapiVersion,
+        nodeVersion,
+        communityEdition,
+        dependencies,
```

Comment:
I'm not sure I understand why we would return the dependencies?

Reply:
That's a mistake it's a leftover from a previous approach, before I modified the core loader to get installed plugins. I'll remove it